source/braille.py

```diff
@@ -27,73 +27,181 @@
+	# Translators: Displayed in braille for an object which is a
+	# grouping.
+	controlTypes.ROLE_GROUPING: _("grp"),
+	# Translators: Displayed in braille for an object which is a
+	# embedded object.
+	controlTypes.ROLE_EMBEDDEDOBJECT: _("embedded"),
+	# Translators: Displayed in braille for an object which is a
+	# end note.
+	controlTypes.ROLE_ENDNOTE: _("enote"),
+	# Translators: Displayed in braille for an object which is a
+	# foot note.
+	controlTypes.ROLE_FOOTNOTE: _("fnote"),
+	# Translators: Displayed in braille for an object which is a
+	# terminal.
+	controlTypes.ROLE_TERMINAL: _("term"),
+	# Translators: Displayed in braille for an object which is a
+	# section.
+	controlTypes.ROLE_SECTION: _("sect"),
+	# Translators: Displayed in braille for an object which is a
+	# toggle button.
+	controlTypes.ROLE_TOGGLEBUTTON: _("tbtn"),
+	# Translators: Displayed in braille for an object which is a
+	# split button.
+	controlTypes.ROLE_SPLITBUTTON: _("sptbtn"),
+	# Translators: Displayed in braille for an object which is a
+	# menu button.
+	controlTypes.ROLE_MENUBUTTON: _("mnubtn"),
+	# Translators: Displayed in braille for an object which is a
+	# spin button.
+	controlTypes.ROLE_SPINBUTTON: _("spnbtn"),
+	# Translators: Displayed in braille for an object which is a
+	# tree view button.
+	controlTypes.ROLE_TREEVIEWBUTTON: _("tvbtn"),
+	# Translators: Displayed in braille for an object which is a
+	# menu.
+	controlTypes.ROLE_MENU: _("mnu"),
+	# Translators: Displayed in braille for an object which is a
+	# panel.
+	controlTypes.ROLE_PANEL: _("pnl"),
+	# Translators: Displayed in braille for an object which is a
+	# password edit.
+	controlTypes.ROLE_PASSWORDEDIT: _("pwdedt"),
 }
 
 positiveStateLabels = {
-	# Translators: Displayed in braille when an object (e.g. a check box) is checked.
-	controlTypes.STATE_CHECKED: _("(x)"),
-	# Translators: Displayed in braille when an object (e.g. a check box) is half checked.
-	controlTypes.STATE_HALFCHECKED: _("(-)"),
 	# Translators: Displayed in braille when an object is selected.
 	controlTypes.STATE_SELECTED: _("sel"),
-	# Translators: Displayed in braille when an object has a popup (usually a sub-menu).
-	controlTypes.STATE_HASPOPUP: _("submnu"),
-	# Translators: Displayed in braille when an object supports autocompletion.
-	controlTypes.STATE_AUTOCOMPLETE: _("..."),
-	# Translators: Displayed in braille when an object (e.g. a tree view item) is expanded.
-	controlTypes.STATE_EXPANDED: _("-"),
-	# Translators: Displayed in braille when an object (e.g. a tree view item) is collapsed.
-	controlTypes.STATE_COLLAPSED: _("+"),
+	# Displayed in braille when an object (e.g. a toggle button) is pressed.
+	controlTypes.STATE_PRESSED: u"⢎⣿⡱",
+	# Displayed in braille when an object (e.g. a check box) is checked.
+	controlTypes.STATE_CHECKED: u"⢎⣿⡱",
+	# Displayed in braille when an object (e.g. a check box) is half checked.
+	controlTypes.STATE_HALFCHECKED: u"⢎⣭⡱",

```

Comment:
I'm a little concerned this looks a bit like a Latin x, so users transitioning might be confused. Since we're doing ⢎⣿⡱ for checked, how about ⢎⡇⡱ or ⢎⢸⡱ for half?